src/main/resources/templates/slurm/command/sbatch

```diff
@@ -0,0 +1,29 @@
+sbatch
+[# th:if="${envVariables != null}"]
+    --export "ALL,[(${envVariables})]"
+[/]
+[# th:if="${not options.useAllEnvVars && envVariables != null}"]
+    --export "[(${envVariables})]"
+[/]
+[# th:if="${options.useAllEnvVars && envVariables == null}"]
+    --export=ALL
+[/]
+[# th:if="${options.priority != null && options.priority != 0}"]
+    --priority=[(${options.priority})]
+[/]
+[# th:if="${options.name != null}"]
+    -J[(${options.name})]
```

Comment:
Let's use a long version of the flag (`--job-name`) here and for working directory (`--chdir`).
It will make code more readable, and we will have a single style of flags specification

Reply:
done